src/js/modules/rpc/server.ts

```diff
@@ -63,6 +66,27 @@ export class ProcessBatchServer extends SupervisorServer {
     return Promise.resolve({ responses });
   }
 
+  disable_coprocessors(
+    input: DisableCoprosRequest
+  ): Promise<DisableCoprosReply> {
+    const responses = input.ids.map<DisableCoprocessorData>((id) => {
+      try {
+        const [handle] = this.repository.getHandlesByCoprocessorIds([id]);
```

Comment:
Do we need to think about any synchronization? Incoming records for this coprocessor may still be arriving as we are removing the element from the store.